### PR TITLE
Fe coverphoto gallery

### PR DESCRIPTION
**Pull request by gursharan4312**
### What this PR does (required):
- New UI for profile photo and cover photo update
- Add Gallery to edit section for users to upload images
- Added Lightbox to show images

### Screenshots / Videos (required):

https://user-images.githubusercontent.com/35828532/121310431-3a256900-c8b8-11eb-8c84-e0b8eef0ae9a.mp4

image
image
image

### CODE REVIEW

**Comment by gursharan4312** [comment]
New Changes:

- Added separate section for Gallery
image

File: server/middleware/error.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ exports.errorHandler = (err, req, res, next) => {
   //  set locals, only providing error in development
   res.locals.message = err.message;
   res.locals.error = req.app.get('env') === 'development' ? err : {};
-
+  console.log(err.message);
```
Comment (rajivtitus) [inline server/middleware/error.js]: Remove

File: client/src/components/Account/ProfilePhoto/ProfilePhoto.tsx [hunk 2]
```diff
@@ -39,110 +32,201 @@ export default function UploadPhoto(): JSX.Element {
...
+              anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
+              transformOrigin={{ vertical: 'top', horizontal: 'center' }}
+            >
+              <MenuItem className={classes.menuItem} onClick={() => handleNewImageButtonClick('cover')}>
+                <ListItemIcon>
+                  <AddAPhotoIcon />
+                </ListItemIcon>
+                <ListItemText primary="New Cover Image" />
+              </MenuItem>
+              <MenuItem className={classes.menuItem} onClick={() => handleDeletePhoto('cover')}>
```
Comment (rajivtitus) [inline client/src/components/Account/ProfilePhoto/ProfilePhoto.tsx]: Just a small suggestion, maybe we can disable 'Delete Cover Image / Profile Photo' if the user does not have any photos in their profile?

**Review by rajivtitus — APPROVED** [review]
UI is fluid and looks great, good job! A suggestion for UI, you can add extra margins to the gallery images to space them out a bit more maybe?

I just made a couple of comments, please have a look.

**Comment by gursharan4312** [comment]
> UI is fluid and looks great, good job! A suggestion for UI, you can add extra margins to the gallery images to space them out a bit more maybe?
> 
> I just made a couple of comments, please have a look.

Changes done

**Pull request merged**